gnome-initial-setup/pages/endless-eula/gis-endless-eula-page.c

```diff
@@ -419,8 +130,12 @@ gis_endless_eula_page_constructed (GObject *object)
 static void
 gis_endless_eula_page_locale_changed (GisPage *page)
 {
+  GisEndlessEulaPage *gis_ee_page = GIS_ENDLESS_EULA_PAGE (page);
+  GisEndlessEulaPagePrivate *priv = gis_endless_eula_page_get_instance_private (gis_ee_page);
+
   gis_page_set_title (page, _("Terms of Use"));
-  reload_terms_view (GIS_ENDLESS_EULA_PAGE (page));
+  if (priv->eula_viewer != NULL)
```

Comment:
In what circumstances could this be `NULL`?

Reply:
I had the case of this being `NULL` when initializing the page. I'll double-check what happened there.